src/ntfs/filter.rs

```diff
@@ -144,21 +144,14 @@ fn filter_from_idxset<T: Id<T>>(
     });
 }
 
-fn updated_stop_time_attributes<T>(
+fn updated_stop_time_attributes(
```

Comment:
I have the feeling that this entire function could be rewritten in 1 line.

```rust
fn updated_stop_time_attributes(
    vehicle_journeys: &CollectionWithId<VehicleJourney>,
    attributes_map: HashMap<(String, u32), String>,
) -> HashMap<(String, u32), String> {
    attributes_map.retain(|((vj_id, _), _)| vehicle_journeys.contains_key(&vj_id));
    attributes_map
}
``` 

or maybe even shorter
```rust
fn updated_stop_time_attributes(
    vehicle_journeys: &CollectionWithId<VehicleJourney>,
    attributes_map: &mut HashMap<(String, u32), String>,
) {
    attributes_map.retain(|((vj_id, _), _)| vehicle_journeys.contains_key(&vj_id));
}
```